pages/common/parallel-lint.md

```diff
@@ -0,0 +1,28 @@
+# parallel-lint
+
+> A tool to check the syntax of PHP files in parallel.
+> More information: <https://github.com/JakubOnderka/PHP-Parallel-Lint>.
+
+- Lint a specific directory:
+
+`parallel-lint {{path/to/directory}}`
+
+- Lint a directory with the specified number of parallel processes:
+
+`parallel-lint -j {{processes}} {{path/to/directory}}`
+
+- Lint a directory, excluding the specified directory:
+
+`parallel-lint --exclude {{path/to/excluded_directory}} {{path/to/directory}}`
+
+- Lint a directory of files using a comma-separated list of extension(s):
```

Comment:
```
Lint a directory of files using a comma-separated list of PHP related extension(s):
```

Reply:
It can be any extensions. You can define `-e go` if you wanted to. 👍 Personally, I'm not sure it's really necessary to mention that they are PHP related.

Reply:
Same here. Though listing extensions is a bit odd, and not really common practice. Is it a glob-based system, or a literal list of extensions? Perhaps a concrete example might be better here.

Reply:
I see. 
I thought it was only for PHP files because the description is specific about PHP.
@sbrl I think it is a literal list

> `-e <ext>` Check only files with selected extensions separated by comma. (default: php,php3,php4,php5,phtml,phpt)

Reply:
Yeah, it's a literal list of extensions that are comma-separated (as mentioned by @andrik). 👍 I'll update the example.